packages/expo-cli/src/credentials/views/AndroidKeystore.ts

```diff
@@ -136,12 +146,20 @@ class DownloadKeystore implements IView {
   constructor(private experienceName: string, private options?: DownloadKeystoreOptions) {}
 
   async open(ctx: Context): Promise<IView | null> {
-    const { confirm } = await prompt({
-      type: 'confirm',
-      name: 'confirm',
-      message: 'Do you want to display the Android Keystore credentials?',
-      when: () => this.options?.displayCredentials === undefined && !this.options?.quiet,
-    });
+    let confirm;
+
+    if (ctx.nonInteractive) {
+      confirm = true;
+    } else {
+      const result = await prompt({
+        type: 'confirm',
+        name: 'confirm',
+        message: 'Do you want to display the Android Keystore credentials?',
+        when: () => this.options?.displayCredentials === undefined && !this.options?.quiet,
+      });
+
+      confirm = result.confirm;
+    }
```

Comment:
```suggestion
    let confirm;

    if (this.options?.displayCredentials !== undefined || this.options?.quiet) {
      confirm = this.options?.displayCredentials ?? !this.options?.quiet
    } else if (ctx.nonInteractive) {
      confirm = true;
    } else {
      const result = await prompt({
        type: 'confirm',
        name: 'confirm',
        message: 'Do you want to display the Android Keystore credentials?',
      });

      confirm = result.confirm;
    }
```
I'm not 100% sure, so double check if it make sense